dist/src/main/resources/application-auth-oidc.yaml

```diff
@@ -0,0 +1,17 @@
+spring:
```

Comment:
🔧 I think that the typical place for these values is in `application-<profile>.properties` - I don't think there is anything functionally different between the two but I think its good to use a single approach here if possible.

Reply:
Do you mean to switch from yaml to properties? I find it more readable in cases like this that there is a multi level hierarchy.

Reply:
Yea so elsewhere in the code we have things like `application-identity.properties` for all the identity related things. NABD but I wanted to raise that it goes against the current approach

Reply:
I'll apply your comment in my next PR (loading claims in authentication object when login with oidc)